_data/directories.yml

```diff
@@ -26,8 +26,6 @@ websites:
 - name: ELIPAY
   url: https://elipay.com/
   img: elipay.png
-  twitter: eligmacom
```

Comment:
the twitter account for this still exists. The facebook though does not. Please put the twitter account back.